README.md

```diff
@@ -25,6 +25,11 @@ SCRIP is a very old format not maintained anymore but is still the most effectiv
 ESMF is able to basically handle any netCDF file that follows the CF-conventions version 1.6 and includes lat/lon values and corners.
 This means that ESMF mesh files are also able to describe unstructured grids.
 
+**Hint:** Once a static file generation is started, it is best
+practice to **not anymore copy or move the repository**. This ensures
+tracability of absolute paths, e.g. absolute paths of map files saved
+in the domain file as NetCDF attributes.
+
```

Comment:
Moving/copying/cloning on the same system is still fine.

I still think any path that does not go out of the repo's working directory are great and even improves reproducibility when the repo is moved out of the system (e.g. a different remote).

I think the hint is too much.  I don't really want to read this.  I probably won't move the repo anyway, but if I do, I will likely have forgotten about the hint.

On the other hand, considering the netCDF attributes, I see the advantage of the absolute path.  Is the system name also stored in the netCDF attributes (e.g. `:Source =`)?  Then I'm convinced that this is OK.